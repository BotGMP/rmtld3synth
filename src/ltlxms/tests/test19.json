["Equal", ["Proposition", "a"], ["Proposition", "b"]]