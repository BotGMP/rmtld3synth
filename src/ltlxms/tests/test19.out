(and (forall ((xs Real) (ys Real)) (=> a b))
     (forall ((xs Real) (ys Real)) (=> b a)))
